evm/p2p/peer.py

```diff
@@ -109,10 +117,13 @@ class BasePeer:
     # FIXME: Must be configurable.
     listen_port = 30303
 
-    def __init__(self, remote, privkey, reader, writer, aes_secret, mac_secret,
-                 egress_mac, ingress_mac, chaindb, network_id):
+    def __init__(self, remote: Node, privkey: datatypes.PrivateKey, reader: asyncio.StreamReader,
```

Comment:
This is really hard to read, can we do one per line for these arguments?

Reply:
Sure!